index.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as Handler from './types/handler';
+import * as Options from './types/options';
+
+export = errorHandler;
+
+declare function errorHandler(
+  options?: Options.ErrorHandlerOptions
+): errorHandler.StrongErrorHandler
+
+declare namespace errorHandler {
```

Comment:
Do we need this namespace? Isn't it enough to export `Handler` under the name `ErrorHandler`?

```ts
import {Handler as ErrorHandler} from './types/handler';
import {ErrorHandlerOptions} from './types/options';

export {ErrorHandler, ErrorHandlerOptions};
```

Reply:
Your code would have worked if exported functions were properties of `module.exports`. Instead, our module exports `createStrongErrorHandler` as `module.exports` itself, so the export above won't work.

I don't understand everything here (like the `export = errorhandler` syntax) but I've managed to get it to work as expected following this template: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html